Azure/ARO-RP
# Remove montioring controller to allow setting persistence

### Which issue this PR addresses:


Fixes https://issues.redhat.com/browse/ARO-13325

### What this PR does / why we need it:

Enables configuration of persistence for the monitoring stack

### Test plan for issue:

1. Create cluster
2. Deploy ARO operator with changes
3. Try and edit monitoring configmap
4. Verify config is not removed
5. Verify ARO operator logs related to removing config are absent. 

### Is there any documentation that needs to be updated for this PR?
Probably yes

### How do you know this will function as expected in production? 

Testing
